cupy/cuda/compiler.py

```diff
@@ -61,7 +61,8 @@ def _run_cc(cmd, cwd, backend, log_stream=None):
                 env['PATH'] = path
         log = subprocess.check_output(cmd, cwd=cwd, env=env,
                                       stderr=subprocess.STDOUT,
-                                      universal_newlines=True)
+                                      universal_newlines=True,
+                                      creationflags=(subprocess.CREATE_NO_WINDOW if _win32 else None))
```

Comment:
The default value looks `0` here 😄
https://docs.python.org/3/library/subprocess.html#subprocess.Popen
```suggestion
                                      creationflags=(subprocess.CREATE_NO_WINDOW if _win32 else 0))
```

Reply:
Ah good catch, thanks!